From ac03b41ee6c1ebe12110dbcbbb15bb2ea4a3d1f2 Mon Sep 17 00:00:00 2001
From: padmashree mandri <padmashree.mandri@intel.com>
Date: Tue, 6 Aug 2024 11:01:46 +0000
Subject: [PATCH] Remove stub HAL for audio

This patch adds changes to disbale stub module for audio
which is causing boot issue, as it is disabled by aosp by
default in A15

Tracked-On: OAM-123104
Signed-off-by: padmashree mandri <padmashree.mandri@intel.com>
---
 groups/device-specific/caas/framework_manifest.xml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index f50d97e..de47548 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -80,11 +80,6 @@
         <version>2</version>
         <fqname>IConfig/default</fqname>
     </hal>
-    <hal format="aidl">
-        <name>android.hardware.audio.core</name>
-        <version>1</version>
-        <fqname>IModule/stub</fqname>
-    </hal>
     <hal format="aidl">
         <name>android.hardware.audio.core</name>
         <version>1</version>
-- 
2.45.2

